include/spirv-tools/optimizer.hpp

```diff
@@ -475,6 +475,12 @@ Optimizer::PassToken CreatePrivateToLocalPass();
 // and computations with constant operands.
 Optimizer::PassToken CreateCCPPass();
 
+// Creates a workaround driver bugs pass.  This pass attempts to work around
+// known driver bugs by identifying the bad code sequences and rewriting them.
+//
+// Current workarounds: avoid OpUnreachable instructions in loops.
+Optimizer::PassToken CreateWorkaroundPass();
```

Comment:
Bikeshedding: I'd like this to be more specific than just "workaround"
I'd rather have  "CreateWorkaround1209Pass"

We may end up with 5 workaround that have various subsets targeted at different particular drivers.  So keeping it specific and separable is preferable.  Even if the name is embarrassing.